core/src/main/scala/org/apache/spark/memory/UnifiedMemoryManager.scala

```diff
@@ -130,6 +130,8 @@ private[spark] class UnifiedMemoryManager private[memory] (
 
 object UnifiedMemoryManager {
 
+  private val RESERVED_SYSTEM_MEMORY_BYTES = 300 * 1024 * 1024
```

Comment:
you should document what this is and update the classdoc for UnifiedMemoryManager